transformations/negated_antonym_perturbation/transformation.py

```diff
@@ -0,0 +1,40 @@
+import nltk
+from nltk.corpus import wordnet
+import nltk.tokenize as nt
+
+from interfaces.SentenceOperation import SentenceOperation
+from tasks.TaskTypes import TaskType
+
+''' Class that converts adjectives and adverbs to its negated antonym'''
+
+
+class NegatedAntonym(SentenceOperation):
+    tasks = [TaskType.TEXT_CLASSIFICATION, TaskType.TEXT_TO_TEXT_GENERATION]
```

Comment:
You can add tasks SENTIMENT_ANALYSIS, PARAPHRASE_DETECTION, TEXTUAL_ENTAILMENT here.